extractor.rb

```diff
@@ -0,0 +1,6 @@
+class Extractor
+  def method_that_does_the_things
+    some_really_complex_code
```

Comment:
Ben, Ben, Ben, back at it again with the overcomplicated code😞. Is this even necessary? You just don’t want to simplify things!!!🙄